packages/fuselage-hooks/README.md

```diff
@@ -101,6 +103,20 @@ Hook to catch which responsive design' breakpoints are active.
 
 Returns **[Array](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array)&lt;[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)>** an array of the active breakpoint names
 
+### UseClipboardParams
+
+Hook to copy the passed content to the clipboard.
```

Comment:
This is wrong due to a misplaced doc comment.

Reply:
New docs generated